package.json

```diff
@@ -42,6 +43,7 @@
     "ipfs-core": "0.3.0",
     "ipfs-http-client": "48.1.1",
     "jest": "^26.6.3",
+    "js-ipfs-ceramic": "git+https://github.com/ceramicnetwork/js-ipfs-ceramic.git",
```

Comment:
this package should be deprecated in favor of `@ceramicnetwork/ipfs-daemon`

Reply:
Yup can change now that it's working